src/spam-protection/get-google-recaptcha-token.ts

```diff
@@ -0,0 +1,15 @@
+import { ReadableDataStore } from '@bigcommerce/data-store';
+
+import { InternalCheckoutSelectors } from '../checkout';
+
+import CardingProtectionActionCreator from './carding-protection-action-creator';
+
+export default function getGoogleRecapthaToken(store: ReadableDataStore<InternalCheckoutSelectors>, error: any) {
```

Comment:
There's a typo here, it should say `RecaptchaToken` rather than `RecapthaToken`

Reply:
`error` should be typed as `unknown` if you don't know the type at this point. This forces you to write and apply a type guard before using it.